compileoptions.cmake

```diff
@@ -66,6 +66,8 @@ if(CLR_CMAKE_PLATFORM_UNIX_ARM)
      add_definitions(-DARM_SOFTFP)
      add_compile_options(-mfloat-abi=softfp)
      add_compile_options(-target armv7-linux-gnueabi)
+   elseif(CLR_CMAKE_PLATFORM_ALPINE_LINUX)
+     add_compile_options("-march=armv7-a")
```

Comment:
A nit - can you please remove the double quotes to make it consistent with the other options we set?

Reply:
Fixed.

Reply:
Couldn't reopen after the bad rebase. New PR https://github.com/dotnet/coreclr/pull/17730.